### DISCUSSION TITLE
Add academic title to Author Schema

### DISCUSSION THREAD

**Discussion by rgaiacs** [quarto-dev/quarto-cli#6050] [Feature Requests]
### Description

I would like to add academic title (like Doctor) to [Author Schema](https://quarto.org/docs/journals/authors.html#author-schema). The reason for this addition is that in a multi-author blog, some authors would like to have their academic title displayed.

**Comment by mcanouil** [comment]
Closing in favour of #6138.

**Comment by arnaudgallou** [comment]
Possibly related, some journals seem to also use actual titles in addition to the academic degree. For example, The Lancet mentions `Prof` before some names in the affiliation statement. See [here](https://www.thelancet.com/action/showPdf?pii=S0140-6736%2823%2900806-1). I don't know if that's used in other similar journals but you might know better.
